docs/datasets_and_tables/gtfs_rt.md

```diff
@@ -0,0 +1,66 @@
+# GTFS Realtime
+
+We collect realtime data every 20 seconds for feeds listed in [agencies.yml](../warehouse/02_agencies.md).
+This data is processed and validated daily.
+
+
+## Data
+
+| dataset | description |
+| ------- | ----------- |
+| `gtfs_rt` | Internal warehouse dataset for preparing GTFS RT views |
+| `views.gtfs_rt_*` | User-friendly tables for analyzing GTFS RT data  |
+| `views.validation_rt_*` | User-friendly tables for analyzing GTFS RT validation data |
+
+## View Tables
+
+Note that this data is still a work in progress, so no views have been created yet.
+
+| Tablename | Description | Notes |
+|----- | -------- | -------|
+| | | |
+
+## Internal Tables
+
+| Tablename | Description | Notes |
+| --------- | ----------- | ----- |
+| `calitp_files` | Metadata on each RT feed file we sample every 20 seconds (e.g. vehicle positions) | |
+| `vehicle_positions` | One row per feed individual vehicle position reported every 20 seconds | Sampling times occur in 20 second intervals, but not on specific points in time. |
+| `validation_service_alerts` | Each row is the contents of an individual results file from the GTFS RT validator | |
+| `validation_trip_updates` | Similar to above, but for trip updates. | |
+| `validation_vehicle_positions` | Similar to above, but for vehicle positions. | |
+
+## Maintenance
```

Comment:
Everything from this header to the end seems to be more about airflow stuff. I recently made a new `airflow` folder in the docs that I would like to advocate to be the place for all airflow-related items. The information about datasets is still helpful to include here, but the stuff about airflow maintenance is a little less relevant to people not doing stuff with airflow.

So, maybe this section could be in a new file at `airflow/realtime-data-pipeline.md`.

Reply:
Ah--I think the payments dataset docs also have maintenance sections that discuss airflow. Is it okay if we merge these in, and create an issue for moving all of this info out (for all datasets; the main other one is payments)?

I can go through and 

* copy out / remove maintenance sections across all files in datasets into the airflow section.
* replace them in each dataset with links to the airflow maintenance docs

Reply:
Added an issue here, since it stacks well with some other docs changes requested: https://github.com/cal-itp/data-infra/issues/783

Reply:
Sounds good to do as part of another PR.